commit/merkleroot/rmn/controller.go

```diff
@@ -174,19 +174,19 @@ func (c *controller) ComputeReportSignatures(
 		}
 	}
 
-	minObserversMap, err := c.rmnHomeReader.GetMinObservers(rmnRemoteCfg.ConfigDigest)
+	homeFMap, err := c.rmnHomeReader.GetF(rmnRemoteCfg.ConfigDigest)
 	if err != nil {
-		return nil, fmt.Errorf("get min observers: %w", err)
+		return nil, fmt.Errorf("get f: %w", err)
 	}
 	// Filter out the lane update requests for chains without enough RMN nodes supporting them.
 	for chain, l := range updatesPerChain {
-		if _, exists := minObserversMap[cciptypes.ChainSelector(chain)]; !exists {
-			return nil, fmt.Errorf("no min observers for chain %d", chain)
+		if _, exists := homeFMap[cciptypes.ChainSelector(chain)]; !exists {
+			return nil, fmt.Errorf("no f for chain %d", chain)
 		}
-		if l.RmnNodes.Cardinality() < minObserversMap[cciptypes.ChainSelector(chain)] {
+		if l.RmnNodes.Cardinality() < homeFMap[cciptypes.ChainSelector(chain)]+1 {
```

Comment:
I think we can re-use the consensus helpers here which are more expressive, `consensus.FPlus1(homeFMap[chain])`

Reply:
Could we have a helper in the `consensus` package that does that using generics?